Dream-Home-PEABOT/FEE_DreamHome
# Om feature display error page

### What does this PR do?
- It creates an Error component
- It styles the Error component
- It adds missing paths to components in `App.tsx`
### Where should the reviewer start?
- `src/components/Error/Error.css`
- `src/components/Error/Error.tsx`
- `src/components/App/App.tsx`

### How should this be manually tested?
- run npm start
- write a nonexisting path to the web browser, e.g `http://localhost:3001/nonexistingpath` and the error page should be visible

### Any background context you want to provide?
### What are the relevant tickets?
closes #51 
### Questions:

### Screenshots (if appropriate)
<img width="702" alt="Screen Shot 2021-01-08 at 9 27 20 PM" src="https://user-images.githubusercontent.com/56229864/104083283-95b0d500-51fa-11eb-856b-01b0a4ac6d61.png">
